hrc/hrc/proto.hrc

```diff
@@ -125,6 +125,10 @@
     <location link="base/csharp.hrc"/>
     <filename>/\.cs$/i</filename>
   </prototype>
+  <prototype name="fsharp" group="main" description="F#">
```

Comment:
Sorry, slipped off from my radar. I am thinking that introducing new languages to the list may shift automatic keyboard shortcuts, and also that the list of main languages should be updated with http://www.tiobe.com/tiobe-index/

Kotlin is not counted there. Well, ok. I guess it is an issue for another bug report for release. )